recipes/scikit-build/python27-x64.patch

```diff
@@ -0,0 +1,13 @@
+MSVC 9 x64 is broken on appveyor. Use NMake there.
```

Comment:
Yeah, my thinking is we will install those patches in the CI beforehand as they are valuable occasionally. However, trying to use Microsoft Visual C++ Compiler for Python 2.7 is preferable.